storage/src/backend/connection.rs

```diff
@@ -355,26 +374,29 @@ impl Connection {
     }
 
     /// Send a request to server and wait for response.
-    pub fn call<R: Read + Send + 'static>(
+    /// `requesting_auth` is true means this request is dedicated for authentication, which takes
+    /// some special process when the request is not authorized. IOW, status code UNAUTHORIZED is returned.
+    /// If this this authentication dedicated request, but `auth_through` is disabled, directly pass
```

Comment:
That means:
If the auth_through is enable, all requests are send to the mirror server.
If the auth_through disabled, e.g. P2P/Dragonfly, we try to avoid sending non-authorization request to the mirror server,  which causes performance loss. `requesting_auth` means this request is to get authorization from a server, which must be a non-authorization request. IOW, only the `requesting_auth` is false and the headers contain authorization token, we can send this request to mirror.

Reply:
> That means: If the auth_through is enable, all requests are send to the mirror server. If the auth_through disabled, e.g. P2P/Dragonfly, we try to avoid sending non-authorization request to the mirror server, which causes performance loss. `requesting_auth` means this request is to get authorization from a server, which must be a non-authorization request. IOW, only the `requesting_auth` is false and the headers contain authorization token, we can send this request to mirror.

This comment LGTM, how about updating it?

Reply:
That's a good idea. I have updated it.